### PR TITLE
Merge To main

### PR DESCRIPTION
**Pull request by Rita2023**
Please check before merging! we are finalize all the main functions

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,59 @@
-# CITS5505_Group_Project
-This is the repository for CITS5505 Group Project
+# Orange Book Web Application
+
+## Description
+The Orange Book web application is a platform designed to capture and share the positive energy and beautiful lives of the youth in this era. Users can create an account and log in to share their experiences through text and image posts. The application also allows users to update profiles, follow or unfollow other users, search and more. It aims to foster a supportive and inspiring community where young people can connect and celebrate their achievements and everyday moments.
```
Comment (Rita2023) [inline README.md:4]: That is a great description of our project

File: README.md [hunk 2]
```diff
@@ -1,2 +1,59 @@
...
+1. **Clone the repository**:
+    ```bash
+    git clone https://github.com/JOEY-WANG-UWA/CITS5505_Group_Project.git
+    cd CITS5505_Group_Project
+    ```
+
+2. **Create and activate a virtual environment**:
+    ```bash
+    python -m venv venv
+    source venv/bin/activate  # On Windows use `venv\Scripts\activate`
```
Comment (Rita2023) [inline README.md:26]: It is nice to have consideration

File: microblog/app/models.py [hunk 3]
```diff
@@ -54,6 +55,12 @@ class User(UserMixin, db.Model):
     def __repr__(self):
         return '<User {}>'.format(self.username)
 
+    def avatar_url(self):
+        if self.avatar:
+            return url_for('static', filename='img/avatar/' + self.avatar)
```
Comment (Rita2023) [inline microblog/app/models.py:60]: The file path is now correct

File: microblog/app/static/img/logo1.png
Comment (Rita2023) [inline microblog/app/static/img/logo1.png:1]: This new logo is nicer for different backgroud

**Review by Rita2023 — COMMENTED** [review]
Thanks for the hard wroks

File: README.md
Comment (Zihan-Zhang810) [inline README.md:1]: Well done

File: microblog/app/models.py [hunk 4]
```diff
@@ -146,51 +153,7 @@ def verify_reset_password_token(token):
         'Upload', back_populates='user')
 
 
-class SearchableMixin(object):
-    @classmethod
```
Comment (JOEY-WANG-UWA) [inline microblog/app/models.py:150]: Search method has already been modified as SQLITE
Previously is ELASTICSEARCH

**Review by JOEY-WANG-UWA — APPROVED** [review]
Approved!!!!!!!

File: microblog/app/models.py [hunk 5]
```diff
@@ -54,6 +55,12 @@ class User(UserMixin, db.Model):
     def __repr__(self):
         return '<User {}>'.format(self.username)
 
+    def avatar_url(self):
```
Comment (Zihan-Zhang810) [inline microblog/app/models.py:58]: The addition of the avatar_url method is a good enhancement for managing user avatars.

File: microblog/app/static/css/style.css
Comment (Zihan-Zhang810) [inline microblog/app/static/css/style.css:1]: Good styles!!

**Review by Zihan-Zhang810 — APPROVED** [review]
Approved!! Well done!!

**Pull request merged**